backend/pick-git/src/main/java/com/woowacourse/pickgit/post/domain/PostRepository.java

```diff
@@ -12,7 +12,7 @@
 
     Optional<Post> findByUser(User user);
 
-    @Query("select p from Post p left join fetch p.user order by p.id")
+    @Query("select p from Post p left join fetch p.user order by p.createdAt desc")
     List<Post> findAllPosts(Pageable pageable);
 
     @Query("select p from Post p where p.user = :user "
```

Comment:
```suggestion
    List<Post> findAllPostsByUserOrderByCreatedAtDesc(User user, Pageable pageable);
```

이 부분 JPQL 안 쓰고 suggestion처럼 메소드명으로 처리할 수 있어요!

Reply:
메서드 네임으로도 가능하긴 한데, 해당 방식을 적용하게 되면 다음과 같은 한계가 있어 최대한 지양하려고합니다.

* 로직 변경이 있을 때마다 메서드 네임이 일괄적으로 수정되어야 함 (테스트 코드 포함)
* 메서드 네임 자체가 너무 길어져서 가독성을 해칩니다.
  * 아직 정렬 조건이 확정난게 아니라서, 이보다 로직이 더 심화되는 경우 메서드 네임으로는 한계가 있습니다.
* 아무리 batch size를 적용하더라도 추후 join이나 fetch join을 활용할 가능성이 많은 부분입니다.